pages/instructorQuestionSettings/instructorQuestionSettings.ejs

```diff
@@ -84,6 +84,30 @@
               <th>Assessments</th>
               <td><% if (locals.assessments) { %><%- include('../partials/assessments', {assessments: assessments}); %><% } %></td>
             </tr>
+            <tr>
+              <th>Sharing Sets</th>
+              <td>
+                <% sharing_sets_in.forEach(function(sharing_set) { %>
+                <%= sharing_set.name %>,
+                <% }); %>
+                <form name="sharing-set-add-<%= question.id %>" method="POST" class="d-inline">
```

Comment:
It shouldn't be necessary to include the question ID here, unless there's a specific reason that I missed?

```suggestion
                <form name="sharing-set-add" method="POST" class="d-inline">
```

Reply:
done